assets/sass/utils/rtl/ltr.scss

```diff
@@ -5,5 +5,11 @@ $left: left;
 $right: right;
 
 @import "rtl-sass";
-@import "test";
-@import "../../style";
+@import "../../dependencies";
+// Hack: Include top level styles outside of namespace
+@import "../../1_basics/base";
```

Comment:
@middle8media this is a bit hacky, because base has top level html and body styles.
Ideally base would be clearly separated from other basic styles.